plugins/lighthouse/src/api.ts

```diff
@@ -85,13 +85,32 @@ export interface Website {
 /** @public */
 export type WebsiteListResponse = LASListResponse<Website>;
 
+/** @public */
+export enum FormFactor {
+  Mobile = 'mobile',
+  Desktop = 'desktop',
+}
```

Comment:
We really try to avoid using `enum`; it's a pretty gnarly part of typescript. We'd typically do `export type FormFactor = 'mobile' | 'desktop'` instead.